src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/_params.py

```diff
@@ -0,0 +1,11 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands import register_cli_argument
+
+register_cli_argument('billing invoice list', 'generate_url', options_list=('--generate-download-url', '-g'), action='store_true', required=False, help='generate download url of the invoice')
```

Comment:
Usually we use `-g` to represent `--resource-group`. Using `-g` could potentially be confusing.